### PR TITLE
Added usecases and zarr mentions

### PR DESCRIPTION
**Pull request by marynaggita**

### CODE REVIEW

File: _data/marynaggita/README.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+- https://twitter.com/latlong_blog/status/1543726888908009472
+- https://twitter.com/_jhamman/status/1579882167303168004
+- https://twitter.com/ErikvanSebille/status/1578030942144188417
+
+
+
+## Zarr Data Visualisation
+
+Atmospheric Conditions from the Coupled Model Intercomparison Project Phase 6 (CMIP6)(https://www.wdc-climate.de/ui/cmip6?input=CMIP6.CMIP.AS-RCEC.TaiESM1.1pctCO2)
```
Comment (sanketverma1704) [inline _data/marynaggita/README.md]: Is there a data visualisation we can see here?

File: _data/marynaggita/README.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+
+## Zarr Data Visualisation
+
+Atmospheric Conditions from the Coupled Model Intercomparison Project Phase 6 (CMIP6)(https://www.wdc-climate.de/ui/cmip6?input=CMIP6.CMIP.AS-RCEC.TaiESM1.1pctCO2) 
+
+
+## Code Snippets
+
+- NASA Prediction of Worldwide Energy Resources (POWER) - https://registry.opendata.aws/nasa-power
```
Comment (sanketverma1704) [inline _data/marynaggita/README.md]: This code has already been posted in the repository. Can you please try to find another code snippet or maybe create one on your own? 😄

**Comment by sanketverma1704** [comment]
Hi @marynaggita. Thanks for sending the PR.
I've left some comments, please have a look at them.

**Comment by marynaggita** [comment]
@MSanKeys963 , I have updated my readme.md. Please review and advise

File: _data/marynaggita/README.md [hunk 3]
```diff
@@ -0,0 +1,220 @@
...
+- https://twitter.com/_jhamman/status/1579882167303168004
+- https://twitter.com/ErikvanSebille/status/1578030942144188417
+
+
+
+## Zarr Data Visualisation
+
+![image1](https://github.com/marynaggita/beautiful-zarr/blob/marynaggita/_data/marynaggita/Screenshots/data3.png)
+
+Use Case: Study Amazon Estuaries with Data from the EOSDIS Cloud(https://github.com/podaac/tutorials/blob/master/notebooks/SWOT-EA-2021/Estuary_explore_inCloud_zarr.ipynb)
```
Comment (sanketverma1704) [inline _data/marynaggita/README.md]: Is this the description of the first image or the second one?

File: _data/marynaggita/README.md [hunk 4]
```diff
@@ -0,0 +1,220 @@
...
+machine urs.earthdata.nasa.gov
+    login <your username>
+    password <your password>
+
+Make sure that this file is only readable by the current user or you will receive an error stating "netrc access too permissive."
+
+$ chmod 0600 ~/.netrc
+
+You will be prompted for your username and password if you dont have a netrc file. Note: these imports are all in the Python 3.6+ standard library.
+
```
Comment (sanketverma1704) [inline _data/marynaggita/README.md]: May I know what's the use of this text here?

**Comment by sanketverma1704** [comment]
> @MSanKeys963 , I have updated my readme.md. Please review and advise

Thanks for updating the PR, @marynaggita. I've added two comments, please have a look at them.

File: _data/marynaggita/README.md (hunk 3 above)
Comment (marynaggita) [reply _data/marynaggita/README.md]: > Is this the description of the first image or the second one?

the description is for the second one.

File: _data/marynaggita/README.md (hunk 4 above)
Comment (marynaggita) [reply _data/marynaggita/README.md]: > May I know what's the use of this text here?

apologies. I thought it was necessary.  Let me keep it away.

Can i add another use case?

**Comment by marynaggita** [comment]
@MSanKeys963  Kindly review the new changes and advise

File: _data/marynaggita/README.md [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+- https://twitter.com/ErikvanSebille/status/1578030942144188417
+
+## Zarr Use Cases
+- Day's worth of analysis files for a single variable and combine them using high-level APIs.(https://mesowest.utah.edu/html/hrrr/zarr_documentation/html/xarray_one_day_analysis_example.html)
+
+- Analyzing MOM6 with a python/xarray/dask/zarr software stack (https://www.cesm.ucar.edu/events/2020/MOM6/files/2020-MOM6-Dussin.pdf)
+
+- Test ZARR performance for CAB-LAB data cubes (https://notebook.community/CAB-LAB/cablab-cubeio/notebooks/zarr_cube)
+
+- Beyond netCDF: Cloud Native Climate Data with Zarr and XArray(https://ui.adsabs.harvard.edu/abs/2018AGUFMIN33A..06A/abstract)
```
Comment (sanketverma1704) [inline _data/marynaggita/README.md:16]: This link throws a 404. Can you please add the correct link here?

File: _data/marynaggita/README.md [hunk 6]
```diff
@@ -0,0 +1,55 @@
...
+
+## Zarr Data Visualisation
+- Boundaries of active fires (red outlines) estimated using VIIRS 375-m thermal anomalies, and smoke from wildfires in the Pacific Northwest on 9 Sep 2020 (https://worldview.earthdata.nasa.gov/)
+
+<img src="https://github.com/marynaggita/beautiful-zarr/blob/marynaggita/_data/marynaggita/Screenshots/data4.jpg" width="400" height="400" display="block" margin= "0 auto">
```
Comment (sanketverma1704) [inline _data/marynaggita/README.md]: This visualisation looks excellent, but the data visualised here is not in Zarr format. It's in `HDF-EOS` format Can you please remove it or add another Zarr data visualisation? Thanks!

**Comment by sanketverma1704** [comment]
Thanks for updating the PR, @marynaggita. I've added some comments above. Please have a look.

File: _data/marynaggita/README.md (hunk 5 above)
Comment (marynaggita) [reply _data/marynaggita/README.md:16]: @MSanKeys963  its a valid working link on my side. 
Cant tell as to why you are getting  a 404

**Comment by marynaggita** [comment]
@MSanKeys963 , I have edited my readme. For the use cases, i cant tell as to why the link is not showing up on your side

File: _data/marynaggita/README.md (hunk 6 above)
Comment (marynaggita) [reply _data/marynaggita/README.md]: I have removed it. Thanks so much

File: _data/marynaggita/README.md (hunk 5 above)
Comment (sanketverma1704) [reply _data/marynaggita/README.md:16]: It works now. My bad.

File: _data/marynaggita/README.md (hunk 6 above)
Comment (sanketverma1704) [reply _data/marynaggita/README.md]: Thank you.

**Comment by sanketverma1704** [comment]
Thanks for updating the PR. It looks good now. Merging this. Feel free to add more stuff as you go.

**Pull request merged**